### PR TITLE
Loader tweaks

### PR DESCRIPTION
**Pull request by daviddrysdale**
# Checklist

- [ ] Pull request affects core Oak functionality (e.g. runtime, SDK, ABI)
  - [ ] I have written tests that cover the code changes.
  - [ ] I have checked that these tests are run by [Cloudbuild](cloudbuild.yaml)
  - [ ] I have updated [documentation](docs/) accordingly.
  - [ ] I have raised an [issue](https://github.com/project-oak/oak/issues) to
        cover any TODOs and/or unfinished work.
- [ ] Pull request includes prototype/experimental work that is under
      construction.

### CODE REVIEW

File: oak/server/rust/oak_loader/BUILD [hunk 1]
```diff
@@ -23,6 +23,9 @@ package(
 rust_binary(
     name = "oak_loader",
     srcs = glob(["src/**/*.rs"]),
+    crate_features = [
+        "oak_debug",
```
Comment (ipetr0v) [inline oak/server/rust/oak_loader/BUILD:27]: Will it be possible to enable this feature from command line (i.e. from `./scripts/run_server`) ?

File: oak/server/rust/oak_loader/src/main.rs [hunk 2]
```diff
@@ -78,7 +74,11 @@ fn read_file(filename: &str) -> Result<Vec<u8>, Box<dyn std::error::Error>> {
 }
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    simple_logger::init_by_env();
+    if cfg!(feature = "oak_debug") {
+        simple_logger::init_by_env();
+    } else {
+        eprintln!("No debugging output configured at build time");
```
Comment (ipetr0v) [inline oak/server/rust/oak_loader/src/main.rs:80]: Does it mean that all logs will be suspended? Even `INFO`

File: oak/server/rust/oak_loader/BUILD (hunk 1 above)
Comment (daviddrysdale) [reply oak/server/rust/oak_loader/BUILD:27]: No, deliberately &ndash; from the hash of the built binary it should be possible to be convinced that there can be no information leak via logging.

File: oak/server/rust/oak_loader/src/main.rs (hunk 2 above)
Comment (daviddrysdale) [reply oak/server/rust/oak_loader/src/main.rs:80]: Yep, anything that could leak private information is compiled out.

**Pull request merged**